sdcm/cluster_aws.py

```diff
@@ -829,9 +835,14 @@ def node_config_setup(self, node, seed_address=None, endpoint_snitch=None, murmu
 
         node.config_setup(**setup_params)
 
-    def node_setup(self, node, verbose=False, timeout=3600):
-        endpoint_snitch = self.params.get('endpoint_snitch')
-        seed_address = ','.join(self.seed_nodes_ips)
+    def _scylla_pre_install(self, node):
+        # Assume that all nodes in the cluster use the same AMI. So, need to check the availability of Scylla once.
+        if self._need_to_install_scylla is None:
+            self._need_to_install_scylla = node.get_scylla_version() is None
```

Comment:
this is broken, it's not always tries to install scylla from scratch, and fails

Reply:
fixed in #1843